app/javascript/admin/controllers/config_controller.js

```diff
@@ -415,20 +415,6 @@ export default class ConfigController extends Controller {
     this.authSectionFormTarget.submit();
   }
 
-  activateMissingKeysModal(providers) {
```

Comment:
@juliannatetreault this should not be removed. This gets called from 

```
if (this.enabledProvidersWithMissingKeys().length > 0) {
      event.preventDefault();
      this.activateMissingKeysModal(this.enabledProvidersWithMissingKeys());
    } else {
```
and if you trace this back with `enabledProvidersWithMissingKeys` you'll see that its related to validating the authentication provider keys